src/components/postCard/menu.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import Menu from "@mui/material/Menu";
+import MenuItem from "@mui/material/MenuItem";
+import MoreVertIcon from "@mui/icons-material/MoreVert";
```

Comment:
Can export all 'mui' imports from a separate file and import from a single file.

Reply:
Updated